03_launch_mgmt_cluster.sh

```diff
@@ -276,6 +263,18 @@ function launch_cluster_api_provider_metal3() {
       kubectl scale -n metal3 deployment.v1.apps capm3-controller-manager --replicas 0
       nohup make run >> capm3.out.log 2>> capm3.err.log &
     fi
+
+    if [ "${BMO_RUN_LOCAL}" = true ]; then
```

Comment:
Not directly related to this PR but I can see both `=` and `==` being used in this file. Not sure which one is correct but we can use one convention IMO

Reply:
I think both `=` and `==` are same, so maybe, we can agree to use one convention?

Reply:
you could try to stick with what is prevalent in the file

Reply:
I have updated all conditions with `==` for the sake of using one convention everywhere